packages/demo/usecase-demo/src/components/NavBar/LeftNavBar.tsx

```diff
@@ -19,6 +19,7 @@ export function LeftNavBar() {
     { defaultValue: false }
   );
   const { isDevMode, setDevMode } = useDevModeStore();
+  const [menuOpen, setMenuOpen] = useState(false);
```

Comment:
Let's try to follow convention to always start with `is` for a boolean value. I propose:

```suggestion
  const [isMenuOpen, setMenuOpen] = useState(false);
```